extensions/amp-story/1.0/pagination-buttons.css

```diff
@@ -125,53 +119,43 @@ amp-story.i-amphtml-story-desktop-one-panel .i-amphtml-story-button-hidden {
 .i-amphtml-story-desktop-fullbleed .i-amphtml-story-button-move {
   height: 48px !important;
   width: 48px !important;
-  margin: auto 12px !important;
+  margin: auto 0 !important;
   background-color: transparent !important;
-  background-size: auto !important;
-  filter: drop-shadow(0 0 16px rgba(0, 0, 0, 0.5)) !important;
+  filter: invert(1) drop-shadow(0 0 6px #000)!important;
   opacity: 1 !important;
 }
 
+.prev-container > .i-amphtml-story-button-move,
+[dir=rtl] .i-amphtml-story-fwd-next > .i-amphtml-story-button-move,
+.i-amphtml-story-fwd-next > .i-amphtml-story-button-move,
+[dir=rtl] .prev-container > .i-amphtml-story-button-move {
+  background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="48" height="48" fill="none"><path stroke="#000" stroke-linecap="round" stroke-linejoin="round" stroke-width="2.4" d="m21.52 16.22 7.75 7.75M21.52 31.75 29.27 24"/></svg>')!important;
+}
+
 .prev-container > .i-amphtml-story-button-move,
 [dir=rtl] .i-amphtml-story-fwd-next > .i-amphtml-story-button-move {
-  background-image: url('data:image/svg+xml;charset=utf-8,<svg width="16" height="25" xmlns="http://www.w3.org/2000/svg"><path d="m15.7 22-9.5-9.5L15.7 3l-3-3L.4 12.6 12.8 25"/></svg>')!important;
-  background-position: 45% 50%!important;
   left: 0!important;
   right: auto !important;
+  transform: rotate(180deg);
 }
 
-.i-amphtml-story-desktop-fullbleed .prev-container > .i-amphtml-story-button-move,
-.i-amphtml-story-desktop-fullbleed[dir=rtl] .i-amphtml-story-fwd-next > .i-amphtml-story-button-move {
-  background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="48" height="48" viewBox="0 0 24 24" fill="#FFF"><path d="M11.7 3.9 9.9 2 0 12l9.9 9.9 1.8-1.8L3.5 12z"/><path fill="none" d="M0 0h24v24H0z"/></svg>') !important;
-  background-position: 10px 0 !important;
```

Comment:
This might be changing the look of the arrows on supports-landscape. Confirming that this is desired:
<img width="925" alt="Screen Shot 2021-12-06 at 3 16 39 PM" src="https://user-images.githubusercontent.com/3860311/144916154-3ec84c7d-f756-4f44-9c65-4a3fede1e9ac.png">

Reply:
Yes, we have a new icon for those arrows (and the refresh icon as well). If you're talking about the size, I'll check with Hong on these details.

Reply:
We'll make them somewhat larger than what it currently is (will end up in a similar size to the pewvious version). WIll fix and commit the changes

Reply:
Updated, and adding this image to the PR description as well
![image](https://user-images.githubusercontent.com/22420856/145259335-24ab1acc-98f0-4e6a-b9f1-6bc76c8574fb.png)